daemon/main.go

```diff
@@ -811,6 +812,66 @@ func runDaemon() {
 	log.Info("Launching node monitor daemon")
 	go d.nodeMonitor.Run(path.Join(defaults.RuntimePath, defaults.EventsPipe), bpf.GetMapRoot())
 
+	if err := d.EnableK8sWatcher(5 * time.Minute); err != nil {
+		log.WithError(err).Fatal("Unable to establish connection to Kubernetes apiserver")
+	}
+
+	log.Info("waiting until all pre-existing policies have been received")
+	d.k8sResourceSyncWaitGroup.Wait()
```

Comment:
Should we also wait if there was not endpoints restored? Also, should we also wait if the user didn't chose restore state?

Reply:
Yes, we need to wait as well. Otherwise we allow creating new endpoints without any policy while k8s already has policies loaded.